dynamo/prediction/fate.py

```diff
@@ -477,16 +486,21 @@ def fate_bias(
         cur_t, n_steps = t[i], len(t[i])
 
         # ensure to identify sink where the speed is very slow if inds is not provided.
-        # if inds is the percentage, use the last percentage of steps to check for cell fate bias.
+        # if inds is set, use inds
+        # if use_sink_percentage is set, then calculate avg_speed and sink_checker
```

Comment:
please add that the speed_percentile is used to determine the time indicies for calculating the fate bias